tests/system/python/packages/test_multiple_assets.py

```diff
@@ -0,0 +1,308 @@
+        # Total number of assets that would be created
+        total_assets = PER_BENCHMARK_ASSET_COUNT * total_benchmark_services
+
+        for count in range(total_benchmark_services):
+            service_name = BENCHMARK_SOUTH_SVC_NAME + "{}".format(count + 1)
+            add_benchmark(fledge_url, service_name, count + 1)
+            verify_service_added(fledge_url, service_name)
+
+        # Wait until total_assets are created
+        time.sleep(PER_BENCHMARK_ASSET_COUNT + 2 * wait_time)
+        verify_ping(fledge_url)
+        verify_asset(fledge_url, total_assets)
+
+        tracking_details = utils.get_asset_tracking_details(fledge_url, "Ingest")
+        assert len(tracking_details["track"]), "Failed to track Ingest event"
+        assert total_assets == len(tracking_details["track"])        
+        for service_count in range(total_benchmark_services):            
+            for asset_count in range(PER_BENCHMARK_ASSET_COUNT):                
+                service_name = BENCHMARK_SOUTH_SVC_NAME + "{}".format(service_count + 1)
+                asset_name = ASSET_NAME + "-{}{}".format(service_count + 1, asset_count + 1)
+                assert service_name in [s["service"] for s in tracking_details["track"]]
+                assert asset_name in [s["asset"] for s in tracking_details["track"]]
+                assert "Benchmark" in [s["plugin"] for s in tracking_details["track"]]
+
+        put_url = "/fledge/restart"
+        put_request(fledge_url, urllib.parse.quote(put_url))
+        # Wait for fledge to restart
+        time.sleep(wait_time * 3)
+
+        # We are adding more total_assets number of assets
+        total_assets = total_assets * 2
+
+        for count in range(total_benchmark_services):
+            service_name = BENCHMARK_SOUTH_SVC_NAME + "{}".format(count + 4)
+            add_benchmark(fledge_url, service_name, count + 4)
+            verify_service_added(fledge_url, service_name)
+
+        # Wait until total_assets are created
+        time.sleep(PER_BENCHMARK_ASSET_COUNT + 2 * wait_time)
+        verify_ping(fledge_url)
+        verify_asset(fledge_url, total_assets)
+
+        tracking_details = utils.get_asset_tracking_details(fledge_url, "Ingest")
+        assert len(tracking_details["track"]), "Failed to track Ingest event"
+        assert total_assets == len(tracking_details["track"])        
+        for service_count in range(total_benchmark_services * 2):            
+            for asset_count in range(PER_BENCHMARK_ASSET_COUNT):                
+                service_name = BENCHMARK_SOUTH_SVC_NAME + "{}".format(service_count + 1)
+                asset_name = ASSET_NAME + "-{}{}".format(service_count + 1, asset_count + 1)
+                assert service_name in [s["service"] for s in tracking_details["track"]]
+                assert asset_name in [s["asset"] for s in tracking_details["track"]]
+                assert "Benchmark" in [s["plugin"] for s in tracking_details["track"]]
```

Comment:
this block duplication can be avoid with a def with total_benchmark_services arg pass.